lib/src/Uncertainty/Algorithm/MetaModel/LinearModel/openturns/LinearModelAlgorithm.hxx

```diff
@@ -48,9 +48,9 @@ public:
                        const Sample & outputSample);
 
   /** Parameters constructor */
-  LinearModelAlgorithm(const Sample & inputSample,
-                       const Basis & basis,
-                       const Sample & outputSample);
+  LinearModelAlgorithm(const Sample &inputSample,
+                       const Sample &outputSample,
+                       const Basis &basis);
 
```

Comment:
I think we need a backward-compatible path (eg use this in persalys)

Reply:
yes

Reply:
ok

Reply:
When do you plan to drop the oldest ctor ? 1.20 or 1.21?

Reply:
one release overlap is enough for this one, so 1.21